### ISSUE TITLE
Failed to dectect location 

### ISSUE THREAD

**Issue by silver1590** [SamBoKing/Secret#2]
As of March 31 2017 pokemon go has been unable to detect any location I set in fly gps

**Comment by DarthXavier311** [comment]
Yep mine too I even commented on the fgps app but get nothing I've researched it , nothing to fix it ... I've messed with fgps settings my own gps and it worked for like 5 seconds starting loading stuff finally and , it stopped... have no idea what is going on

**Comment by SawsanIris** [comment]
Please... 
If you find the way to make it work again write it here

**Comment by Aakash30Sethiya** [comment]
I have android oreo in my phone but it still. When I run the mock location it shows error "Failed to detect location(12)".

**Comment by ArctikWolf98** [comment]
I'm having the same issue 4years later. How has it not been fixed yet?
